jstz_node/src/tailed_file.rs

```diff
@@ -0,0 +1,44 @@
+use std::io::SeekFrom;
```

Comment:
Is there much point of re-implementing the traits?

Could we just have a newtype that is `BufReader<File>` that implements `Deref` and `DerefMut`.

https://doc.rust-lang.org/stable/core/ops/trait.Deref.html

Reply:
good point, idt we even need the Deref/DefreMut here. we can just reimplement the lines method